azalea-protocol/src/packets/game/clientbound_player_info_update_packet.rs

```diff
@@ -196,6 +196,6 @@ impl McBufWritable for ActionEnumSet {
         if self.update_display_name {
             set.set(5);
         }
-        set.write_into(buf)
+        set.as_u8().write_into(buf)
```

Comment:
It's supposed to already have this behavior if there's at most 8 bits. This seems more like a bug with writing bitsets.